lithops/standalone/worker.py

```diff
@@ -58,8 +59,13 @@ def run_worker(master_ip, job_key):
         resp = requests.get(url)
 
         if resp.status_code != 200:
-            logger.info('All tasks completed'.format(url))
-            return
+            if STANDALONE_CONFIG.get('exec_mode') == 'reuse':
+                logger.debug('All tasks completed, waiting for new tasks'.format(url))
+                time.sleep(1)
+                continue
+            else:
+                logger.info('All tasks completed'.format(url))
+                return
```

Comment:
Re-usability in the worker should be always activated, even in the same job (create mode). In some circumstances, this change will prevent a single map job to finish and will make lithops waiting forever for a job completion.

Reply:
In what circumstances it may prevent job to finish?

Reply:
1. Create mode starts as many VMs as elements in iterdata. Gen 2 is limited to max 200vCPU (100VMs if using 2vpcu flavor), so if someone overcomes this quota limit and the worker is not able to process more than one task per job, the job will never be completed and lithops will be stuck waiting forever.
2. Sometimes a I experienced that a VM is never started, so a task will be never unprocessed.

Some time ago I added work queues in `master.py`, and allowed workers to process more than one task of a job, to prevent these issues, and ensure a single map job is always executed with the available workers.

Reply:
This PR actually comes to resolve the described issue as well. Currently we need to have N workers for N tasks. 
But, when the mode is "reuse", instead of pushing to the "queue per worker", all tasks are pushed to "all" queue. And in case there even a single worker for 100 tasks it will eventually process all by polling one by one from the 'all' queue.

Reply:
So with this PR, in `create` mode jobs will fail again in the cases I described, and users must move to `resuse`?

Reply:
Correct, the `create` mode will not change by this PR. 
We can fix or deprecate the create `mode` later, in a scope of different PR.